casepro/pods/views.py

```diff
@@ -0,0 +1,36 @@
+import json
+from django.http import JsonResponse
+
+from casepro.pods import registry
+
+
+def read_pod_data(request, pk):
```

Comment:
Resolved in https://github.com/rapidpro/casepro/pull/107/commits/5b2f959b9e2d7e96c59506f9476215ba2dffef40